src/js/ko-components/view-models/orifice-calculator.coffee

```diff
@@ -151,13 +154,40 @@ define "orifice-calculator-viewmodel", ["knockout", "lodash", "knockout.validati
           return _.ceil flowRate, 3
 
       @copyFeedbackActive = ko.observable false
-
       @copyFeedbackClass = ko.pureComputed =>
         if @copyFeedbackActive()
           return 'bounce-in'
         else
           return 'hidden'
 
+      # Fields to keep the cookies
+      @orificeBoreDiameterCookies  = ko.observableArray()
+      @baseSpecificGravityCookies  = ko.observableArray()
+      @operatingTemperatureCookies = ko.observableArray()
+      @operatingPressureCookies    = ko.observableArray()
+      @differentialPressureCookies = ko.observableArray()
+
+      @flowRate.subscribe (flowRate) =>
+        if flowRate
+          _.each FIELDS_FOR_SUGGESTION, (field) => @setCookies(field, NUMBER_OF_COOKIES)
+
+      _.each FIELDS_FOR_SUGGESTION, (field) => @initializeFieldWithCookies field
+
+    initializeFieldWithCookies: (variableName) =>
+      cookies = eval(Cookies.get(variableName))
+      if cookies
+        @["#{variableName}Cookies"] _.map cookies, (val) -> { value: val }
+
+    setCookies: (variableName, numberOfCookies) =>
+      cookies = eval(Cookies.get(variableName)) || []
```

Comment:
The value get from the cookies if it is an array will be `"[1,2]"`, so eval is used to change it to `[1,2]`